pkg/kubelet/dockertools/manager.go

```diff
@@ -655,12 +665,25 @@ func (dm *DockerManager) runContainer(
 	return dockerContainer.ID, nil
 }
 
-func setEntrypointAndCommand(container *api.Container, opts *docker.CreateContainerOptions) {
+func setEntrypointAndCommand(container *api.Container, opts *kubecontainer.RunContainerOptions, dockerOpts *docker.CreateContainerOptions) {
```

Comment:
I could see value in moving the expansion logic into ```pkg/kubelet/containers``` and having a function that took a command, and slice of args, and returned back the expanded values.